static/admin/config.yml

```diff
@@ -37,10 +37,26 @@ collections: # A list of collections the CMS should be able to edit
         file: "content/copy/home.yml"
         description: "Copy for the home page"
         fields:
+          - name: homeCallout
```

Comment:
You don't need to qualify this as `home` since this is already in the home copy file